sdk/eventhubs/azure-messaging-eventhubs/test/ut/CMakeLists.txt

```diff
@@ -20,13 +20,14 @@ add_executable (
     checkpoint_store_test.cpp 
     consumer_client_test.cpp    
     event_data_test.cpp
+    eventhubs_admin.cpp
```

Comment:
should `eventhubs_admin` (not `_test`) be here?
If this is a test utility, should it be named something like `eventhubs_admin_test_helper`?

Reply:
I'm calling it `eventhubs_admin_client`, since that's what it is - it's a stripped down management plane client for eventhubs.